bucoffea/scripts/run_quick.py

```diff
@@ -39,14 +48,28 @@ def main():
         # "zjetsnunu_ht1200to2500_mg_2018_v5":[
         # "data/zjetsnunu_ht1200to2500_mg_2018_v5.root"
         # ]
-        "SingleMuon_2018C" : [
-            "./data/SingleMuon_2018C.root"
+        #"SingleMuon_2018C" : [
+        #    "./data/SingleMuon_2018C.root"
+        #]
+        "vbfhinv2017" : [
+            "root://cmsxrootd.fnal.gov//store/user/aandreas/nanopost/21Aug19/VBF_HToInvisible_M125_13TeV_TuneCP5_powheg_pythia8/VBF_HToInvisible_M125_pow_pythia8_2017/190822_134734/0000/tree_1.root"
```

Comment:
1) Remove all the commented lines here.

2) I'd suggest to use the same dataset names everywhere, i.e. change `vbfhinv2017` to `VBF_HToInvisible_M125_pow_pythia8_2017`.  It's better practice to stick to this because sometimes the processors will perform dataset-dependent actions, which rely on parsing of the dataset name.